### PR TITLE
Validation for classification to ensure 2 slashes

### PR DESCRIPTION
**Pull request by pdxjohnny**
Signed-off-by: John Andersen <john.s.andersen@intel.com>

### CODE REVIEW

File: src/probes/telem_record_gen.c [hunk 1]
```diff
@@ -56,6 +56,21 @@ static GOptionEntry options[] = {
         { NULL }
 };
 
+const unsigned int count_chars(const char *check, const char character)
+{
+        unsigned int count = 0U;
+        if (check == NULL) {
+                return count;
+        }
+        for (;*check != '\0'; ++check)
```
Comment (phmccarty) [inline src/probes/telem_record_gen.c]: the first semicolon should be followed by a single space

File: src/probes/telem_record_gen.c [hunk 2]
```diff
@@ -56,6 +56,21 @@ static GOptionEntry options[] = {
...
+        {
```
Comment (phmccarty) [inline src/probes/telem_record_gen.c]: this curly bracket should be moved to the end of the previous line

**Comment by phmccarty** [comment]
I commented on a couple of code style convention issues.

Otherwise, looks good to me.

**Comment by pdxjohnny** [comment]
Let me know if that looks good

**Comment by phmccarty** [comment]
+1

**Pull request merged**